composer.json

```diff
@@ -1,47 +1,52 @@
 {
-    "name": "silverleague/ideannotator",
-    "description": "Generate PHP DocBlock annotations for DataObject and DataExtension databasefields and relation methods",
-    "type": "silverstripe-vendormodule",
-    "keywords": [
-        "silverstripe",
-        "database",
-        "orm",
-        "docblock",
-        "ide",
-        "annotations"
-    ],
-    "license": "BSD-3-Clause",
-    "authors": [
-        {
-            "name": "Martijn van Nieuwenhoven",
-            "email": "info@axyrmedia.nl"
-        },
-        {
-            "name": "Simon 'Firesphere' Erkelens",
-            "email": "simonerkelens@silverstripe.com"
-        }
-    ],
-    "require": {
-        "php": ">=5.6.0",
-        "silverstripe/recipe-cms": "1.0.x-dev",
-        "phpdocumentor/reflection-docblock": "^2.0@dev"
+  "name": "silverleague/ideannotator",
+  "description": "Generate PHP DocBlock annotations for DataObject and DataExtension databasefields and relation methods",
+  "type": "silverstripe-vendormodule",
+  "keywords": [
+    "silverstripe",
+    "database",
+    "orm",
+    "docblock",
+    "ide",
+    "annotations"
+  ],
+  "license": "BSD-3-Clause",
+  "authors": [
+    {
+      "name": "Martijn van Nieuwenhoven",
+      "email": "info@axyrmedia.nl"
     },
-    "require-dev": {
-        "scriptfusion/phpunit-immediate-exception-printer": "^2",
-        "phpunit/phpunit": "^5.7"
+    {
+      "name": "Simon 'Firesphere' Erkelens",
+      "email": "simonerkelens@silverstripe.com"
+    }
+  ],
+  "require": {
+    "php": ">=5.6.0",
+    "silverstripe/recipe-cms": "^1",
```

Comment:
Generally requiring a recipe is a bad idea, it’s better to just require the cms module

Reply:
Good point. Let's see if that works with CI :)